pkg/api/types/container_types.go

```diff
@@ -0,0 +1,28 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package types
+
+type ContainerInspectCommandOptions struct {
+	// GOptions is the global options
+	GOptions GlobalCommandOptions
+	// Format of the output
+	Format string
+	// Inspect mode, either dockercompat or native
+	Mode string
+	// Targets are the containers to be inspected
+	Targets []string
```

Comment:
Maybe just `Containers`?

Reply:
Makes sense, I started off with trying to refactor inspect and used Targets as it could refer to both images or containers. Containers makes sense in this context though. Thanks!

Reply:
Changes made ✅